### DISCUSSION TITLE
SideDownloadError

### DISCUSSION THREAD

**Discussion by BrendaW2024** [aclap-dev/video-downloadhelper#502] [Help and Q&A]
### Your question

Hi

Will VDH ever work again on content on sites that are not public - ie on places like Kajabi where you have to sign in? I've tried the latest Beta on some of this content again in the hopes that it might work but I keep getting side download errors. I know you can't replicate this for content that's not public - but I live in hope...

For the record, this is the error:

{
  "version": "9.0.1.8",
  "target": "google",
  "channel": "beta",
  "page_url": "https://traumahealingaccelerated.mykajabi.com/biology-of-trauma-online-summit",
  "error": {
    "details": "SideDownload error: : ",
    "downloadable_id": "downloadable_7423693232776487",
    "error": "coapp_failure",
    "id": "coapp_failure_f643ad20-4252-4c9b-81e7-ab4791d5bcc3"
  }
}

Thanks for your help

### Addon & CoApp version and details

version: 9.0.1.8
target: google
channel: beta
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"2d532c5775e048e6"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/brendawille/Move onto Hard Drive",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 5
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user, like you.

Being behind a paywall does not automatically mean VDH won't work on that site.  I've been on pay sites where VDH worked just fine.  So the problematic factor is not the paywall.  There is a problematic factor but it's not the paywall.

There is something you could try.  No guarantees but we won't know unless we try.  I explain it in detail over here: #485.
